Segfault

This test
```
2. Testing conversion between the grid types:
  Extracting cell corner nodes from a double-density grid...done
  Extracting centre nodes from corner nodes...done
  Recovering corner nodes from centre nodes..../test.sh: line 28:   856 Segmentation fault      (core dumped) ../getnodes gridpoints_CE.txt -i CE -o CO > gridpoints_CO2.txt
make: *** [all] Error 139
```

is segfaulting :unamused: 

Not sure what is going on there and the tested passed in the `staged-recipes` step.

--- a/recipe/build.sh
+++ b/recipe/build.sh
@@ -9,9 +9,11 @@ pushd gridutils
 ./configure --prefix=$PREFIX
 
 make
+
 pushd test
 make
 popd
+
 make install
 
 popd

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -1,10 +1,10 @@
 package:
     name: gridutils
-    version: 1.01.0
+    version: 1.0
 
 source:
     git_url: https://github.com/sakov/gridutils-c.git
-    git_tag: master  # upstream does not create releases :-(
+    git_tag: v1.0
 
 build:
     number: 0

